rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/directives/AnyParamDirectives.scala

```diff
@@ -14,9 +14,17 @@ trait AnyParamDirectives {
         provide(FormData.Empty)
     }
 
+  private def extractAwsXRayTracingHeader(request: HttpRequest): Map[String, String] = {
+    request.headers.find(_.name() == "X-Amzn-Trace-Id").map(header => header.name() -> header.value()).toMap
```

Comment:
I think this should be `.equalsIgnoreCase` as headers are case-insensitive

Reply:
Fixed :)